fatima-krystle_talens-getEven.js

```diff
@@ -0,0 +1,81 @@
+//Get Even
+
+for (var i=0; i<=200; i+=2){
+    console.log(i)
+}
+
+
+//Excited Kitten
+var catSays = ["...human...why you taking pictures of me?...", "...the catnip made me do it...", "meow?", "...why does the red dot always get away..."]
+
+for (var i=1; i<=10; i++){
+    if (i%2===0){
+      console.log(catSays[Math.floor(Math.random()*(catSays.length-1))])}
+    else {console.log("Love me, pet me! HSSSSSS!")}
+}
+
+//Thermostat
+var currentTemp = Math.floor(Math.random()*(100))
+var desiredTemp = 68
+
+console.log("The current temperature is "+currentTemp+"F")
+
+var i=currentTemp
+while (i!==desiredTemp){
+  if (i>desiredTemp){
+    i--;
+    console.log("The current temperature is now "+i+"F")
+    } else if (i<desiredTemp){
+    i++;
+    console.log("The current temperature is now "+i+"F")
+    }
+  }
```

Comment:
looks like you're missing a closing curly bracket for the while loop. also, your indentation looks funky here